### ISSUE TITLE
Incorrect dest folder

### ISSUE THREAD

**Issue by nilooy** [sapphi-red/vite-plugin-static-copy#8]
I have used this config in vite,

```js
    viteStaticCopy({
      targets: [
        {
          src: "./src/test.json",
          dest: 'dist',
        },
      ],
      flatten: true,
    }),
```

but it's being copied on project root nested in full directly tree.
<img width="320" alt="Screenshot 2022-03-28 at 03 32 21" src="https://user-images.githubusercontent.com/32486682/160301899-3ebedecf-c623-4efa-9056-0bf37be26563.png">

**Comment by nilooy** [comment]
Well, just found the issue. 
i was passing `outDir` from  `const outDir = resolve(__dirname, "dist");`
but after removing it, it just work just fine,
Thanks for this awesome plugin 🥇

**Comment by akoidan** [comment]
**vite.config.ts**:
```ts
import { resolve } from "path";
export default {
outDir: resolve(__dirname, '..', 'dist')
root:  resolve(__dirname, '..', 'src');
}
```
[This patch works for me](https://github.com/akoidan/vite-plugin-static-copy/blob/b7be9b44441a2708032d160782410588bf9be945/src/utils.ts#L51)
```diff
diff --git a/src/utils.ts b/src/utils.ts
index 948e337..acf5420 100644
--- a/src/utils.ts
+++ b/src/utils.ts
@@ -48,7 +48,7 @@ export const copyAll = async (
   const copyTargets = await collectCopyTargets(rootSrc, targets, flatten)
   await Promise.all(
     copyTargets.map(({ src, dest }) =>
-      fs.copy(path.join(rootSrc, src), path.join(rootSrc, rootDest, dest))
+      fs.copy(src, path.join(rootDest, dest))
     )
   )
   return copyTargets.length
```

**Comment by akoidan** [comment]
The new fix still doesn't work I get:
```
[vite-plugin-static-copy:build] ENOENT: no such file or directory, lstat '/home/andrew/PycharmProjects/pychat/frontend/src/home/andrew/PycharmProjects/pychat/frontend/node_modules/emoji-datasource-apple/img/apple/64/0023-fe0f-20e3.png'
```

**Comment by sapphi-red** [comment]
@akoidan [This fix](https://github.com/sapphi-red/vite-plugin-static-copy/commit/9526e90f4695149eaec8e7c880f8e42ccd6e8719) is not included in the latest release (v0.4.3).
I have released v0.4.4 which includes the fix. Would you check it again after updating to v0.4.4?

**Comment by sapphi-red** [comment]
Closing as it is currently covered with a test. Please create a new issue with reproduction if it is still not working.
